packages/expo-screen-orientation/package.json

```
@@ -0,0 +1,39 @@
+{
+  "name": "expo-screen-orientation",
+  "version": "1.0.0",
+  "description": "ExpoScreenOrientation standalone module",
+  "main": "build/ScreenOrientation.js",
+  "types": "build/ScreenOrientation.d.ts",
+  "sideEffects": false,
+  "scripts": {
+    "build": "expo-module build",
+    "clean": "expo-module clean",
```

```suggestion
    "clean": "expo-module clean",
    "lint": "expo-module lint",
```
When you rebase with current master, please add a lint script and then run `yarn prepare` to generate eslint config and then `yarn lint` to check whether `expo-screen-orientation` package contains any lint issues.